src/Config/packages.aget

```diff
@@ -9,7 +9,7 @@
       "DynamoVisualProgramming.WpfUILibrary": "2.16.0.2501",
       "DynamoVisualProgramming.ZeroTouchLibrary": "2.16.0.2501",
       "DynamoVisualProgramming.DynamoSamples": "2.16.0.2501",
```

Comment:
as there is no API change compared with 2.16.0, so no change here for these "DynamoVisualProgramming.xxx"